assets/js/pages/HostsList/HostsList.jsx

```diff
@@ -15,6 +15,7 @@ import Table from '@common/Table';
 import Tags from '@common/Tags';
 import Tooltip from '@common/Tooltip';
 
+import { uniqBy } from 'lodash';
```

Comment:
Put this import in line 6 or so (after EOS_WARNING_...).
We put external imports first